sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -4,28 +4,35 @@ Azure Time Series Insights provides data exploration and telemetry tools to help
+- Retrieving and being able to add, update and remove Time Series instances.
+- Retrieving and being able to make changes to the Time Series Insights environment types, such as creating, updating and deleting Time Series types.
+- Retrieving and being able to make changes to the Time Series Insights hierarchies, such as creating, updating and deleting Time Series hierarchies.
+- Executing Time series queries such as querying for raw events, series and aggregate series
+
+[Source Code][tsi_client_src] | [Package (NuGet)][tsi_nuget_package] | [Product documentation][tsi_product_documentation] | [Samples][tsi_samples]
+
+# Key concepts
+
+## TimeSeriesInsightsClient
 
-You can explore the time series insights APIs (using the client library) using the samples project.
+A `TimeSeriesInsightsClient` is the primary interface for developers using the Time Series Insights client library. It provides both synchronous and asynchronous operations to perform operations on a Time Series Insights environment. The `TimeSeriesInsightsClient` exposes several properties that a developer will use to perform specific operations on a Time Series Insights environment. For example, `ModelSettings` is the the property that a developer can use to perform operations on the model settings of the TSI environment. `Instances` can be used to perform operations on TSI instances. Other properties include `Types`, `Hierarchies` and `Query`.
 
-The samples project demonstrates the following:
+# Creating TimeSeriesInsightsClient
 
-- Instantiate the client
-- Demonstrate how to use Model APIs
-    - Create, get, replace, and delete instances
-    - Create, get, replace, and delete types
-    - Create, get, replace, and delete hierarchies
-    - Get and update model configuration settings
-- Demonstrate how to query raw events, series, and aggregates.
+To create a new Time Series Insights client, you need the endpoint to an Azure Time Series Insights instance and supply credentials.
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below,
+or other credential providers provided with the Azure SDK, please install the Azure.Identity package:
 
-## Creating the time series insights client
+```PowerShell
+Install-Package Azure.Identity
+```
 
-To create a new time series insights client, you need the endpoint to an Azure Time Series Insights instance and supply credentials.
-In the sample below, you can set `TsiEndpoint`, `TenantId`, `ClientId`, and `ClientSecret` as command-line arguments.
-The client requires an instance of [TokenCredential](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential?view=azure-dotnet).
-In this samples, we illustrate how to use one derived class: ClientSecretCredential.
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
```

Comment:
Should it be Id or ID?

Reply:
I was wondering that myself yesterday as well :). I thought in code, we use Id. In regular documentation text, we use ID. We can confirm with the team today during the meeting.

Reply:
Not a problem. That's confusing to remember for sure.